test/doctest/with-docinfo-shared.html

```diff
@@ -0,0 +1,14 @@
+<!-- .docinfo-shared -->
```

Comment:
This test shouldn't be provided. The content is invalid. See https://github.com/asciidoctor/asciidoctor-doctest/issues/24.

I suggest you drop it entirely since it will give us a false impression that we do proper docinfo testing.

Reply:
I think it's because I'm using `doctest:generate`. I also had to remove "source-coderay" and "source-pygment" because they fail on JRuby. Can we explicitly exclude these files from the generate command? Or add them to .gitignore? Otherwise we get a "dirty" Git working tree 😞

Reply:
I don't have strong git hygiene, my tree is always dirty so I'm used to it. Not sure how to exclude from generate.

Reply:
> I don't have strong git hygiene, my tree is always dirty so I'm used to it. 

😱 😱 😱 

I have to admit, it makes me feel uncomfortable to have a "dirty" tree 😆 
I will add them in the `.gitignore` with a comment so we can locally test them and we know why the files are ignored. Are you fine with that?

Reply:
`.gitignore` under a section explicitly marked as failing tests would be fine

Reply:
done